tech.feliciano.pocket-casts.yml

```diff
@@ -1,67 +1,83 @@
 app-id: tech.feliciano.pocket-casts
 runtime: org.freedesktop.Platform
-runtime-version: '22.08'
+runtime-version: &runtime-version '24.08'
 sdk: org.freedesktop.Sdk
-sdk-extensions:
-  - org.freedesktop.Sdk.Extension.node18
 base: org.electronjs.Electron2.BaseApp
-base-version: '22.08'
-command: run.sh
+base-version: *runtime-version
+# Add the Node 20 SDK extension.
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+command: start-pocket-casts
 separate-locales: false
 finish-args:
+  # Share IPC namespace with the host, without it the X11 shared memory extension will not work.
```

Comment:
Can you make the changes related to documenting the finish args as a separate commit? This would make the runtime update specific changes clearer.

Reply:
Sure, I can do that.